packages/blade/tsconfig-typecheck.web.json

```diff
@@ -6,7 +6,9 @@
     "**/**.stories.tsx",
     "**/**.stories.internal.tsx",
     "**/**.native.ts",
-    "**/**.native.tsx"
+    "**/**.native.tsx",
+    "**/**.native.test.ts",
```

Comment:
you were facing type errors on the test files before this 🤔

Reply:
Yeah. Only when we check types with `.native` as module suffix